pkg/controller/segment.go

```diff
@@ -0,0 +1,30 @@
+package controller
+
+import (
+	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+type segmentResponse struct {
+	SegmentWriteKey string `json:"segment-write-key"`
```

Comment:
I'm curious - why do we want to use the kebab case for this json key? I would rather prefer using the camelcase for the sake of consistency.

Reply:
this is consistent with what we have in the other files, see here: https://github.com/codeready-toolchain/registration-service/blob/master/pkg/controller/authconfig.go#L11-L14. but i do see that in signup, when the phone verification stuff was added, it brought some inconsistency i guess.

Reply:
I wasn't aware of this file - I was referring to json key format used in all our CRDs...
OK, let's keep it as it is :+1: